recipes/julia/meta.yaml

```diff
@@ -52,7 +52,8 @@ requirements:
 
 test:
   commands:
-    - julia -e 'Base.runtests("all"); Base.runtests("pkg libgit2-online")'
+    - julia -e 'Base.runtests("all"); Base.runtests("pkg libgit2-online")':
+        timeout: 1800s
```

Comment:
I'm trying to do this:

https://stackoverflow.com/questions/36173553/how-to-extend-timeout-for-tests-in-circleci
https://circleci.com/docs/1.0/configuration/#modifiers